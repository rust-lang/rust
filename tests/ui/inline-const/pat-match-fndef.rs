#![feature(inline_const_pat)]

fn uwu() {}

fn main() {
    let x = [];
    match x[123] {
        const { uwu } => {}
        //~^ ERROR `fn() {uwu}` cannot be used in patterns
        _ => {}
    }
}
