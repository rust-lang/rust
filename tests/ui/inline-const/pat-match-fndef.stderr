error: fn item `fn() {uwu}` cannot be used in patterns
  --> $DIR/pat-match-fndef.rs:8:9
   |
LL |         const { uwu } => {}
   |         ^^^^^^^^^^^^^ fn item can't be used in patterns

error: aborting due to 1 previous error

